lib/assertions/is-not-required.js

```diff
@@ -0,0 +1,16 @@
+import elementToString from '../helpers/element-to-string';
+
+export default function notRequired(message) {
+  let element = this.findTargetElement();
+  if (!element) return;
+
+  let result = element.required === false;
+  let actual = element.required === true ? 'required' : 'not required';
```

Comment:
I think you can just reuse the `result` variable here instead of comparing the property again

Reply:
Done. Also updated `is-required.js` to do the same. FWIW, this is what `is-checked` does, so we may want to update that too separately.

Reply:
yeah, agreed. I merged it and planned to do a small cleanup round later :)